lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -14,6 +14,8 @@ const expectations = [
     artifacts: {
       Stacks: [{
         id: 'jquery',
+      }, {
+        id: 'jquery', // for jquery's "fast path" stack. Multiple stacks have a second "fast path" version in the library detector
```

Comment:
remove comment and add `name: 'jQuery (Fast path)'` ?

Reply:
> remove comment and add `name: 'jQuery (Fast path)'` ?

yeah, that might be simpler than my comment idea :)

Reply:
Done :)